news/bugfix-3713.md

```diff
@@ -0,0 +1,10 @@
+The problem was the new python 3.10 isn't compatible with previous "python finding script", because it check is only 1 digit or "<digit>.<digit>" and I added a new condition to check "<digit>.<digit><digit>".
+
+This bug can be easily caused. Only need a python 3.10 version and run the configure with this flag: "--with-python=3.10" it will not find the 3.10, but with the bugfix it will found.
+
+Other possible solution:
+
+- using if-else instead of switch-case, because in the if statement can be put regex expression.
+- using the configure "if-else", because there can put regex expression also.
+
+issue: #3709
```

Comment:
This si going to be part of the news file (an example: https://github.com/syslog-ng/syslog-ng/releases/tag/syslog-ng-3.32.1), if you check out that the description are brief (specially in case of simple bugfix). This description is nice for the PR description, but too detailed for a news entry.

From user point of view the implementation details are not interesting (still they can uncover it as there is a reference to the original PR), I would rather include the fact that configure cannot accept python 3.10 version, and now it does.